tests/sentry/snuba/metrics/test_snql.py

```diff
@@ -434,6 +437,41 @@ def test_dist_count_aggregation_on_tx_satisfaction(self):
             "transaction.tolerated",
         )
 
+    @patch("sentry.models.transaction_threshold.ProjectTransactionThresholdOverride.objects.filter")
```

Comment:
There is a missing test here, can you please create a test that fetches the project transaction threshold once and then examine the contents of the cache and then ensure that the next call gets the value from the cache

Reply:
Yep! Thanks